hadoop-cloud/pom.xml

```diff
@@ -208,6 +208,9 @@
     -->
     <profile>
       <id>hadoop-3.2</id>
```

Comment:
I think we should have an empty profile `hadoop-2.7` to deactivate `hadoop-3.2` when `hadoop-2.7` is set?